node/src/main/java/io/airlift/node/NodeConfig.java

```diff
@@ -169,4 +164,21 @@ public NodeConfig setConfigSpec(String configSpec)
         this.configSpec = configSpec;
         return this;
     }
+
+    public AddressType getAddressType()
+    {
+        return addressType;
+    }
+
+    @Config("node.address-type")
```

Comment:
Maybe `node.internal-address-type`? Since this is for configuring `internal-address`